rtree/index.py

```diff
@@ -299,13 +299,18 @@ def __repr__(self):
                                                               self.get_size())
 
     def __getstate__(self):
-        state = self.__dict__.copy()
-        del state["handle"]
-        return state
+        attrs = self.__dict__.copy()
+        del attrs["handle"]
+        entries = self.intersection(self.bounds, objects=True)
+        entries = [(item.id, item.bbox, item.object) for item in entries]
```

Comment:
When `interleaved=True`, this seems to require `item.bbox`, but when `interleaved=False`, this seems to require `item.bounds`. I'm not even sure why there are two names for the same thing...

Reply:
indeed that seems like some kind of bug due to refactorings. It should be tracked down all the way to figure out what's going on.